src/sp/it/pl/util/system/Environment.kt

```diff
@@ -95,18 +95,18 @@ fun runCommand(command: String, then: Consumer<Process>? = null) {
 
 /**
  * Browse the file in OS' file browser:
- * * if denotes a directory it will be opened
- * * if denotes a file, it's location will be opened and the file selected
+ * - directory -> opn in file browser
+ * - file -> open directory and select file
  *
  * On some platforms the operation may be unsupported. In that case this method is a no-op.
  */
 fun File.browse() = toURI().browse()
 
 /**
  * Browse uri:
- * * if denotes an uri, opens it in its predefined internet browser
- * * if denotes a directory it will be opened
- * * if denotes a file, it's location will be opened and the file selected
+ * - url -> opens it in its standard browser
```

Comment:
I dont like the inconsistent terminology used in this file. Can we pls use 'system associated browser/application/editor' ?

Reply:
yes, will do